src/services/user.js

```diff
@@ -0,0 +1,68 @@
+import { authHeader } from '../utils';
+
+const backend = process.env.VUE_APP_BACKEND;
+
+const handleResponse = (response) => response.text().then((text) => {
+  if (!response.ok) {
+    if (response.status === 401) {
+      return Promise.reject(response.status);
```

Comment:
Instead of requiring the calling code to deal with the `401`, could we do what the article does?
```suggestion
      logout();
      location.reload(true);
```
The reload should detect that there's no token and redirect the user to `/`.

Reply:
Yeah I did that because the linter gets angry when you try to call a function before it gets defined lol

Reply:
We could put `logout()` at the top then?

Reply:
Oh my god, I'm an idiot. I forgot `logout()` doesn't need `handleResponse` lmao. I was about to be like "but then `logout` will complain haha